### ISSUE TITLE
Support: How to solve problems from a specific weekly contest using CLI?

### ISSUE THREAD

**Issue by jan25** [skygragon/leetcode-cli#178]
### Problem Summary

Leetcode has weekly contests. I normally like to solve one of the contests every week(not necessarily the same week's or during a contest. More like a virtual contest of sorts). From my understanding of CLI we have support for all problems on leetcode grouped by difficulty. I was wondering if there is support for problems grouped by weekly contest number for example.

### How to reproduce

NA

### Environment

- leetcode-cli version:

- OS version:
- Node version:
- Npm version:

**Comment by ketankr9** [comment]
No, there isn't any support for it, but it can be implemented, as the leetcode api for weekly contest is open and available as json at [https://leetcode.com/contest/api/info/weekly-contest-150/](https://leetcode.com/contest/api/info/weekly-contest-150/).
Can try it myself during December or later if no one takes initiative by then.

Thanks to @skygragon for such an amazing tool.

**Comment by jan25** [comment]
Thanks @ketankr9! I was not aware of open API or where to find the list of available endpoints. I'll give this a shot as well before then if i'm able to find some time
